clinica/pipelines/statistics_surface/clinica_surfstat.py

```diff
@@ -0,0 +1,157 @@
+from os import PathLike
+from pathlib import Path
+from typing import Dict
+
+import numpy as np
+from brainstat.stats.SLM import SLM
+
+from ._contrasts import _get_contrasts_and_filenames
+from ._inputs import (
+    _build_thickness_array,
+    _extract_parameters,
+    _get_average_surface,
+    _get_t1_freesurfer_custom_file_template,
+    _read_and_check_tsv_file,
+)
+from ._model import _build_model
+from ._outputs import _plot_results, _print_clusters, _save_results
+
+
+def _compute_results(
+    model: SLM,
+    mask: np.ndarray,
+    threshold_uncorrected_pvalue: float,
+    threshold_corrected_pvalue: float,
+) -> Dict:
+    """Take a fitted SLM model and store the results in a dictionary.
+
+    Parameters
+    ----------
+    model : Fitted SLM model.
+    mask : Mask used in the analysis.
+    threshold_uncorrected_pvalue : Threshold used with non corrected P-values.
+    threshold_corrected_pvalue : Threshold used with corrected P-values.
+
+    Returns
+    -------
+    results : Dictionary with the results.
+    """
+    from scipy.stats import t
+
+    results = dict()
+    results["coefficients"] = np.nan_to_num(model.coef)
+    results["TStatistics"] = np.nan_to_num(model.t)
+    results["uncorrectedPValue"] = dict()
+    results["uncorrectedPValue"]["P"] = 1 - t.cdf(results["TStatistics"], model.df)
+    results["uncorrectedPValue"]["mask"] = mask
+    results["uncorrectedPValue"]["thresh"] = threshold_uncorrected_pvalue
+    results["FDR"] = model._fdr()
```

Comment:
PyCharm complains of an access to a protected member here. We are likely supposed to access this information through other means.

I glanced quickly at the implementation of the SLM class, and it looks like this information could be stored in the `model.Q` attribute computed in `model.multiple_comparison_corrections`.